### DISCUSSION TITLE
Feature request:  Multiple instance support

### DISCUSSION THREAD

**Discussion by bwims** [jziolkowski/tdm#88] [Ideas]
Would it be possible to support concurrent multiple instances of tdm so that you could have one instance with one MQTT server and one with another?

**Comment by bwims** [comment]
I think the easiest solution might be to have a way to point to an alternative directory to %USERPROFILE%\TDM for the config files.

**Comment by jziolkowski** [comment]
related to #215

**Comment by bwims** [comment]
Great!  Thanks!

**Comment by jziolkowski** [comment]
released right now, go test it :)

**Comment by bwims** [comment]
On the road right now but will get to it soon!
